### DISCUSSION TITLE
Add support for http on Web App

### DISCUSSION THREAD

**Discussion by leuchtrakete666** [Floorp-Projects/Floorp#968] [Feature Request]
### Pre-Submission Checklist

- [X] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [ ] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [X] I have tried clearing the browser cache and cookies.
- [X] I have tried to create a new profile and the issue still occurs.
- [X] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [x] I have included relevant screenshots or console outputs.
- [ ] I have checked this problem is not ocurring in the latest version of Firefox ESR.
- [X] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

When trying to add a Web-App with unsecure http - http:// - nothing happens, when choosing from the Web-Apps menu. Also they symbol don't appear in the address bar.

### Reproduction Steps

1. Enter an unsecured website address in the address bar. - http://localhost:XXXX
2. The Web-App icon in the address bar is not there, the menu item in Web-Apps don't react, when clicking on it.

### Expected Behavior

The Web-App should be created.

### Actual Behavior

Nothing is created.

### Screenshots

![Screenshot_20240228_103719](https://github.com/Floorp-Projects/Floorp/assets/66916144/df4bce7a-7e28-43f4-b793-29b378262026)

### Browser Version

11.10.5

### Operating System

Manjaro

**Comment by regularentropy** [comment]
Please provide a proper issue name.

**Comment by Ryosuke-Asano** [comment]
http is currently not supported. Convert to feature request

**Comment by yarub123** [comment]
Can confirm, creating a webapp from a (http) local host does NOT work. 

Nonetheless, Floorp is delicious. Best browser in this solar system.

**Comment by creeper-0910** [comment]
Fixed.
